_episodes/03-using-packages-and-channels.md

```diff
@@ -15,27 +14,6 @@ keypoints:
 - "Understand how to use Conda and Pip together effectively."
 ---
 
-## What are Conda packages?

```

Comment:
Perhaps we should leave a link here to say something along the lines of...

```
We have covered the anatomy of Conda Packages in [Section 1](01-getting-started-with-conda).
```
?

Reply:
Hopefully I've dealt with this by imporving the title of episode 3, which is now about how to use channels and work with Conda and PyPI. We should look at ep3 again soon.